### PR TITLE
Fixed compiler warnings.

### PR DESCRIPTION
**Pull request by mikeller**

### CODE REVIEW

File: openLRSng/RFM.h [hunk 1]
```diff
@@ -272,9 +272,8 @@ void rfmInitBeacon()
 
 void rfmClearIntStatus(void)
 {
-uint8_t c;
-  c=spiReadRegister(RFM22B_INTSTAT1);
-  c=spiReadRegister(RFM22B_INTSTAT2);
+  spiReadRegister(RFM22B_INTSTAT1);
+  spiReadRegister(RFM22B_INTSTAT2);
```
Comment (csurf) [inline openLRSng/RFM.h]: oops, forgot to remove this from testing...

Comment (mikeller) [reply openLRSng/RFM.h]: Using `make` gets warnings a lot more into your face. ;-)

File: openLRSng/RX.h [hunk 2]
```diff
@@ -379,7 +379,7 @@ uint8_t bindReceive(uint32_t timeout)
       rfmGetPacket(rxc_buf, len);
 
       if (rxc_buf[0] == 'b') {
-        for (uint8_t i = 0; i < sizeof(bind_data); i++) {
+        for (uint8_t i = 0; i < sizeof(bind_data) && i < sizeof(rxc_buf); i++) {
```
Comment (csurf) [inline openLRSng/RX.h]: didn't catch this. compiler complains about possible overflow ?? bind_data is currently ~20bytes. oh well...

Comment (mikeller) [reply openLRSng/RX.h]: The problem is it can't know the size of `bind_data` at compile time of the file, since it's external.

**Pull request merged**